yt/yt/server/lib/exec_node/config.h

```diff
@@ -664,11 +664,42 @@ DEFINE_REFCOUNTED_TYPE(TNbdConfig)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+DEFINE_ENUM(EJobProxyLoggingMode,
+    (Sandbox)
+    (SeparateDirectory)
```

Comment:
I think that better name these modes as:
* `Simple`
* `SeparateDirectory` or even `PerJobDirectory`